pkg/statdb/inspector.go

```diff
@@ -41,6 +41,10 @@ func (srv *Inspector) CreateStats(ctx context.Context, req *pb.CreateStatsReques
 		AuditSuccessCount:  req.AuditSuccessCount,
 		UptimeCount:        req.UptimeCount,
 		UptimeSuccessCount: req.UptimeSuccessCount,
+		Meta: pb.NodeOperator{
+			Email:  "",
+			Wallet: "",
+		},
```

Comment:
This code change is unnecessary. It just initializes the `Meta` field with its default empty value, which will be done implicitly anyway.